CODEOWNERS

```diff
@@ -1 +1,12 @@
 *       @gdi-python-maintainers @gdi-python-approvers
+
+#####################################################
+#
+# Docs reviewers
+#
+#####################################################
+
+*.md @signalfx/docs
+*.rst @signalfx/docs
+docs/ @signalfx/docs
```

Comment:
@owais I think that you should also add the maintainers and the other approvers here, as per https://github.com/signalfx/gdi-specification/pull/117, so that tech writers reviews don't block PRs.

Reply:
Isn't that covered by line 1? If not, feel free to add in this PR.

Reply:
Added!